src/modules/imports/import.rs

```diff
@@ -69,12 +70,22 @@ impl Import {
     }
 
     fn resolve_import(&mut self, meta: &ParserMetadata) -> Result<String, Failure> {
-        match fs::read_to_string(self.path.value.clone()) {
-            Ok(content) => Ok(content),
-            Err(err) => error!(meta, self.token_path.clone() => {
-                message: format!("Could not read file '{}'", self.path.value),
-                comment: err.to_string()
-            })
+        if self.path.value.starts_with("std") {
```

Comment:
I'd specify this further so that it does not fail when someone tries to import a file named for example `stdout.ab`
```suggestion
        if self.path.value.starts_with("std/") {
```